src/app/tests/suites/certification/Test_TC_SMCO_1_1.yaml

```diff
@@ -0,0 +1,234 @@
+# Copyright (c) 2023 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 4.1.1. [TC-SMCO-1.1] Global Attributes with DUT as Server
+
+PICS:
+    - SMOKECO.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Smoke CO Alarm"
+    endpoint: 1
+
+tests:
+    - label: "Commission DUT to TH"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "TH reads the ClusterRevision attribute from the DUT"
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "TH reads from the DUT the FeatureMap attribute(Smoke Alarm)"
+      PICS: SMOKECO.S.F00
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    - label: "TH reads from the DUT the FeatureMap attribute(CO Alarm)"
+      PICS: SMOKECO.S.F01
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x2]
+
+    - label: "TH reads from the DUT the AttributeList attribute"
+      command: "readAttribute"
+      attribute: "AttributeList"
+      response:
+          constraints:
+              type: list
+              contains: [0, 3, 5, 6, 7]
```

Comment:
OK, so the test plan issue covers the fact that 1 and 2 should not be in the list, but does not mention 4, 8, 9, a, b, c.

Reply:
The test plan uses the description of supported attributes, which I implemented according to spec.

Reply:
The test plan (looking on the right branch, I hope) says:

> Verify that the DUT response contains the AttributeList attribute and have the list of thirteen (13) supported attributes: 0x0000, 0x0001, 0x0002, 0x0003, 0x0004, 0x0005, 0x0006, 0x0007, 0x0008, 0x0009, 0x000a, 0x000b, and 0x000c.

Reply:
I've listed them all, the mandatory attributes together and the optional attributes adding PICS one by one.